examples/tools/reference-lines.md

```diff
@@ -1,6 +1,38 @@
 ---
-layout: needs-example
+layout: tool-simple
```

Comment:
![k6JaT9bPZA](https://user-images.githubusercontent.com/5797588/56087263-3fe8a680-5e35-11e9-9f7e-7b48781a23b8.gif)

Reply:
Perfect, it is done. I am just moving Reference lines to another PR.